src/pages/wheelcalc.tsx

```diff
@@ -0,0 +1,165 @@
+import { Calculator } from '../components/calculator'
+import { Input } from '../components/input'
+
+interface WheelData {
+  numSpins: number
+  numDays: number
+  wheelCredit: boolean
+  runecube: boolean
+}
+
+const DEFAULT_DATA: WheelData = {
+  numSpins: 500,
+  numDays: 7,
+  wheelCredit: true,
+  runecube: true,
+}
+
+export default () => {
+  const [data, values, setValues] = Calculator.useData(DEFAULT_DATA, (settings) => ({
+    numSpins: settings.numSpins ? parseInt(settings.numSpins, 10) : undefined,
+    numDays: settings.numDays ? parseInt(settings.numDays, 10) : undefined,
+    wheelCredit: settings.wheelCredit === undefined ? undefined : !!settings.wheelCredit,
+    runecube: settings.runecube === undefined ? undefined : !!settings.runecube,
+  }))
+
+  // Calculations.
+  const wheelCost = data.wheelCredit ? 5 : 10;
+
+  let coinsTotal = 0;
+  let spinsPerDayString = "";
+  let coinsPerDayString = "";
+		
+  if (data.numSpins % data.numDays !== 0) {
+    let spinsPerDay = Math.floor(data.numSpins / data.numDays);
+    let extraSpinDays = data.numSpins % data.numDays;
+    let regularDays = data.numDays - extraSpinDays;
+    spinsPerDayString =  (spinsPerDay + 1).toLocaleString() + " spins per day for " + extraSpinDays.toLocaleString() + " days, and " +  spinsPerDay.toLocaleString() + " for " + regularDays.toLocaleString() + " days";
```

Comment:
Newer JS (and all TS) supports backticks for string interpolation, like this:

```suggestion
    spinsPerDayString =  `$((spinsPerDay + 1).toLocaleString()} spins per day for ${extraSpinDays.toLocaleString()} days, and ${spinsPerDay.toLocaleString()} for ${regularDays.toLocaleString()} days`
```

Also as a judgement call, you can skip the `toLocaleString()` on things you expect to be "short integers", all locales I know of have no special formatting for ints between 0 and 999. It's fine to leave but sometimes it makes things more readable to leave it out when you know the value will only ever be a small int of some kind.